app/lwip/core/dns.c

```diff
@@ -570,7 +570,7 @@ dns_send(u8_t numdns, const char* name, u8_t id)
   char *query, *nptr;
   const char *pHostname;
   u8_t n;
-  dns_random = os_random()%250;
+  dns_random = os_random();
```

Comment:
since `os_random()` returns an unsigned long, shouldn't `os_random()` be cast to `u16_t`?